tests/whitebox/integration/test_timeout.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2016 Red Hat, Inc.
```

Comment:
Move this entire file to a new directory "tests/whitebox/unit/test_timeout.py". In order for the tests to run, you'll also need to add an empty ```__init__.py``` file in the "tests/whitebox/unittest" directory.